packages/lodestar-cli/src/cmds/account/cmds/validator/deposit.ts

```diff
@@ -96,7 +96,7 @@ export async function handler(options: IAccountValidatorDepositOptions): Promise
       value: (depositData.amount * BigInt(1e9)).toString(),
       data: rlp
     });
-    if (!tx.hash) throw Error("No transaction hash");
+    if (!tx.hash) throw new YargsError("No transaction hash");
```

Comment:
This is not a quite "expected" error so I would throw a regular error

Reply:
done